src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs

```diff
@@ -59,7 +60,7 @@ private void _vm_PropertyChanged(object sender, System.ComponentModel.PropertyCh
                     this.Height = minHeightSize;
                     if (_vm.Children.Count !=0)
                     {
-                        if (_vm.Children[0].NodeLabel.Contains(Environment.NewLine))
+                        if (_vm.Children[0].NodeLabel.Contains(Environment.NewLine) || _vm.Children[0].NodeLabel.ToUpper() == nameof(WatchViewModel.DICTIONARY))
```

Comment:
Could `Children[0]` ever be null?

Reply:
If just a null value is passed to the watch node, i think the watch node doesn't consider it a child and could be zero. Can you verify?

Reply:
> 

In this section of the code when we are dealing with the Iscollection property, no.

Reply:
> Could `Children[0]` ever be null?

In this section of the code when we are dealing with the Iscollection property, no.

Reply:
[QilongTang](https://github.com/QilongTang)  [reddyashish](https://github.com/reddyashish) Please your thoughts/worries, Maybe I'm missing something like others scenarios, complex data types or whatever, It will be nice to test it in order to be on the same page and prevent Dynamo crashes or unexpected behaviors.

Reply:
It would be hard to tell accurately considering all cases. I think you can just add a additional check before that like:
if(_vm.Children.Count > 0 && ....)

Reply:
Oh, I did not see that. Looks good to be merged.

Reply:
In line 63, you can just use the variable NodeLabel you created instead of _vm.Children[0].NodeLabel.